### PR TITLE
generate enc update list csv

### PR DESCRIPTION
**Pull request by MastekRohitChettri**
This is ESS side of  Pbi => 54108 - POS ESS - Upload Batch 5 - Generate "ENC Update List.csv" file in ESS and upload it to FSS

Code changes and Unit tests
Functional Test

### CODE REVIEW

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs [hunk 1]
```diff
@@ -241,6 +243,39 @@ public async Task<bool> CreateProductFile(string batchId, string exchangeSetInfo
             }
         }
 
+        public async Task<bool> CreateEncUpdateCsv(SalesCatalogueDataResponse salesCatalogueDataResponse, string filePath, string batchId, string correlationId)
+        {
+            string file = Path.Combine(filePath, "ENC Updates List.csv");
```
Comment (harshal11869) [inline UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs:248]:
```suggestion
            string file = Path.Combine(filePath, "ENC Update List.csv");
```

**Review by harshal11869 — CHANGES_REQUESTED** [review]

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/appsettings.json [hunk 2]
```diff
@@ -59,6 +59,7 @@
     "InfoFolderAvcsUserGuide": "AVCS-User-Guide.pdf",
     "InfoFolderEnctandPnmstatus": "ENC TandP NM status.pdf",
     "InfoFolderAddsEul": "End-User-Licence-Agreement-for-ADMIRALTY-digital-data-services.pdf",
-    "InfoFolderImpInfo": "Important Information for AVCS users.pdf"
+    "InfoFolderImpInfo": "Important Information for AVCS users.pdf",
+    "EncUpdateList": "ENC Updates List.csv"
```
[the remark this answers is not in the document]
Comment (MastekRohitChettri) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/appsettings.json]: Resolved

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs (hunk 1 above)
Comment (MastekRohitChettri) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs]: Resolved

**Review by harshal11869 — APPROVED** [review]
LGTM

**Review by ajasonca — APPROVED** [review]
Looks good

**Pull request merged**